src/main/resources/ValidationMessages.properties

```diff
@@ -9,8 +9,9 @@ GuestTeacher.locale.required=The locale of the guest teacher is required.
 School.name.required=The name of the school is required.
 School.address.required=The address of the school is required.
 School.postalCode.required=The postal code of the school is required.
-School.contactPersonName.required=The name of the contact person is required.
 School.contactPersonEmail.invalid=The email address of the contact person is invalid.
 School.contactPersonEmail.required=The email address of the contact person is required.
+School.firstName.required=The first name of the contact person is required.
```

Comment:
Arent you forgetting the School.prefix.required for the post DTO?

Reply:
I don't think the prefix should be required.

Reply:
You are 100 percent right.